Cargo.toml

```diff
@@ -140,6 +140,7 @@ bitflags = { version = "2.8.0" }
 blake3 = "1.5.5"
 borsh = { version = "1.5.5", features = ["derive", "unstable__schema"] }
 borsh0-10 = { package = "borsh", version = "0.10.3" }
+boxcar = "0.2.7"
```

Comment:
I know this is coming from an older PR, but any reason to not use the newest version which is committed to the lockfile?

Reply:
done